dashboard/app/models/user.rb

```diff
@@ -381,7 +381,8 @@ def self.name_from_omniauth(raw_name)
         user.age = 21
       else
         # student or unspecified type
-        user.birthday = auth.info.dob
+        age_in_years = ((DateTime.now - Date.parse(auth.info.dob, '%Y-%m-%d')) / 365).floor
```

Comment:
Note that this slightly overestimates the age of a student, as the result of leap years. Using 365.25 could also be an over- or under- estimate. Consequently, I'm accepting slight error here.

If we want to make sure we never overestimate (at the risk of locking out a thirteen year old), let me know, I can change the math to do so.